tests/testthat/test-deprecated.R

```diff
@@ -1,8 +1,20 @@
-
 data(tips)
 
-test_that("ggally-cor", {
+test_that("ggally_cor_v1_5() works", {
+  withr::local_options(lifecycle_verbosity = "quiet")
   expect_silent(
```

Comment:
Let's change this to a `lifecycle::expect_deprecated()` (and remove the expect_silent)

And remove `withr` import as it'll be unused

Reply:
Done! And it's now much better: 7e0adb0bfcb55323bfa444882b4f005ff7274459